torch/csrc/jit/runtime/decomposition_registry.cpp

```diff
@@ -160,6 +160,26 @@ void RegisterDecomposition(
   schema_to_decomposition[&schema] = g;
 }
 
+void run_jit_decomposition(
```

Comment:
Can we delete `run_jit_decomposition` from `functorch/functorch/csrc/BatchRulesHelper.h`?

It is being used elsewhere in functorch but we should deduplicate the code